SPECS/expat/expat.spec

```
@@ -67,13 +67,13 @@ rm -rf %{buildroot}/%{_docdir}/%{name}
 %{_libdir}/libexpat.so.1*
 
 %changelog
+* Thu Mar 28 2024 Aditya Dubey <adityadubey@microsoft.com> - 2.6.2-2
+- Updated changelog to match fasttrack chnagelog 
```

I'd say it's more this:
```suggestion
- Removed unnecessary `-p2` argument
```

This is the change between 2.6.2-1 and 2.6.2-2.